requirements.txt

```diff
@@ -35,6 +35,8 @@ pyvista>=0.32
 pyvistaqt>=0.4
 mffpy>=0.5.7
 ipywidgets
+ipympl
```

Comment:
The only thing I can think of is that adding this dependency could have caused some compatibility issues

Reply:
Ok so my analysis is that this is indeed the issue but I think since the documentation is not being built on `Windows` that we should just skip it. The only difference was installing `ipympl` and `ipyevents` with pip in this PR and the test PR and not doing that in your "Fix CI" PR that just merged and passed all the tests. I think `ipympl` is doing something odd. Maybe we could have the documentation also built on `Windows` without `noplot` but maybe until we actually build it with the plots we should punt this.